### PR TITLE
Log collection on ECS and Docker

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Updates to log collection docs for ECS and Docker

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/docker-ecs-logs/agent/amazon_ecs
https://docs-staging.datadoghq.com/sarina/docker-ecs-logs/agent/amazon_ecs/logs/
https://docs-staging.datadoghq.com/sarina/docker-ecs-logs/agent/docker/log/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by kayayarai — APPROVED** [review]

File: content/en/agent/docker/log.md [hunk 1]
```diff
@@ -81,37 +118,65 @@ The commands related to log collection are:
 | `-e DD_CONTAINER_EXCLUDE="name:datadog-agent"`               | Prevents the Datadog Agent from collecting and sending its own logs and metrics. Remove this parameter if you want to collect the Datadog Agent logs or metrics. This parameter value supports regular expressions. |
 | `-v /var/run/docker.sock:/var/run/docker.sock:ro`     | Logs are collected from container `stdout/stderr` from the Docker socket.                                                                                        |
 
+<<<<<<< HEAD
```
Comment (sarina-dd) [inline content/en/agent/docker/log.md:121]:
```suggestion
```

File: content/en/agent/docker/log.md [hunk 2]
```diff
@@ -81,37 +118,65 @@ The commands related to log collection are:
...
+[1]: /getting_started/agent/autodiscovery?tab=docker
+[2]: https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent
+[3]: /logs/faq/how-to-tail-logs-from-host-using-a-container-agent/
+[4]: https://hub.docker.com/r/datadog/agent/tags
+=======
```
Comment (sarina-dd) [inline content/en/agent/docker/log.md:126]:
```suggestion
```

File: content/en/agent/docker/log.md [hunk 3]
```diff
@@ -81,37 +118,65 @@ The commands related to log collection are:
...
 [1]: https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent
 [2]: https://gcr.io/datadoghq/agent
+>>>>>>> 3670948778edd73c07715993fdee1f425c4c2852
```
Comment (sarina-dd) [inline content/en/agent/docker/log.md:129]:
```suggestion
```

**Review by clamoriniere — APPROVED** [review]
👍

**Comment by sarina-dd** [comment]
@kayayarai just a heads up - not sure why this decided to add link changes for two tracing files after I merged master, but they're in here. 🐋

**Comment by sarina-dd** [comment]
Also, preview is having severe issues. Very different from what I'm seeing locally. working to resolve.

**Comment by sarina-dd** [comment]
Borked tabs resolved! Ready for review @kayayarai + @cohenyair!

**Review by kayayarai — APPROVED** [review]
👍

**Pull request merged**